Assets/Scripts/KeyLocationController.cs

```diff
@@ -65,4 +84,31 @@ private IEnumerator GetPlayers()
             players.AddRange(GameObject.FindGameObjectsWithTag("Player"));
         }
     }
+
+    public void LoseHealth() //Run this when enemies are killed and reach location
+    {
+        health -= 1;
+    }
+
```

Comment:
what is health used for in key location?

Reply:
Enemy # to clear the zone